src/meson.build

```diff
@@ -19,9 +19,16 @@ gdbus_src = custom_target(
 
 daemon_builddir = meson.current_build_dir()
 
+unitdir = ''
+systemd = dependency('systemd', required: false)
+if systemd.found()
```

Comment:
Maybe warn about otherwise?

Reply:
I copied the section from bolt and I've no strong opinion, I'm fine warning or erroring out. Robert any preference?

Reply:
I would make systemd a hard dependency - this package is only going to be used on Ubuntu and so we don't have to handle the case that it's not present.

Reply:
updated